lbz/events/broker.py

```diff
@@ -2,23 +2,26 @@
 from typing import Callable, Dict, List
 
 from lbz.events.event import Event
+from lbz.handlers import BaseBroker
 from lbz.misc import get_logger
 
 logger = get_logger(__name__)
 
 
 # TODO: type_key and data_key will be const for EventBridge and different set for Cognito Events
-class EventBroker:
+class EventBroker(BaseBroker):
     def __init__(
         self,
         mapper: Dict[str, List[Callable]],
```

Comment:
We can make this type more precise I think:
```suggestion
        mapper: Dict[str, List[Callable[[Event], None]]],
```